src/system-health/tests/mock_connector.py

```diff
@@ -2,7 +2,7 @@ class MockConnector(object):
     STATE_DB = None
     data = {}
 
-    def __init__(self, host):
+    def __init__(self, use_unix_socket_path):
```

Comment:
>use_unix_socket_path [](http://example.com/codeflow?start=23&length=20)

Does it need a default value?

Reply:
No, it's just a mock. the variable has no significance when running tests

Reply:
I see the caller code is `SonicV2Connector(use_unix_socket_path=True)`.

So ideally you need to mock for a optional parameter. Even the current works.

Reply:
Still applicable, even this PR is merged.
@liat-grozovik @vivekrnv

Reply:
I've ran the tests with the previous mock, and it failed. That's why i've updated the mock. 

Previously, we used to call `SonicV2Connector(host="127.0.0.1")` with host argument, so the ctor for mock is also expected to have the host argument. Similarly for `use_unix_socket_path` arg